actionlist/app/reducers/action_items.js

```diff
@@ -72,7 +83,80 @@ const toggleDrawer = (state) => {
   };
 };
 
+const selectDropdownOption = (state, action) => {
+  const type = action.payload.option;
+  let newdropdownColors = state.dropdownColors;
+  switch (type) {
```

Comment:
Instead of a lengthy switch, why not do something like 
```es6
return {
  ...newDropdownColors,
  [action.payload.option]: action.payload.value
}
```